packages/sds/src/message_channel/message.ts

```diff
@@ -17,6 +17,7 @@ export class Message implements proto_sds_message.SdsMessage {
     public lamportTimestamp?: bigint | undefined,
     public bloomFilter?: Uint8Array<ArrayBufferLike> | undefined,
     public content?: Uint8Array<ArrayBufferLike> | undefined,
+    public repairRequest: proto_sds_message.HistoryEntry[] = [],
```

Comment:
```suggestion
public causalHistory: proto_sds_message.HistoryEntry[],
public repairRequest: proto_sds_message.HistoryEntry[],
public lamportTimestamp?: bigint | undefined,
public bloomFilter?: Uint8Array<ArrayBufferLike> | undefined,
public content?: Uint8Array<ArrayBufferLike> | undefined
```

might be better to position it after `causalHistory` and before the optionals `?` values.